packages/lib/src/components/internal/StoreDetails/StoreDetails.tsx

```diff
@@ -20,12 +20,7 @@ function StoreDetails({ storeDetails = false, ...props }) {
 
     return (
         <div className="adyen-checkout__store-details">
-            {renderFormField('boolean', {
-                onChange,
-                label: i18n.get('storeDetails'),
-                value,
-                name: 'storeDetails'
-            })}
+            <Checkbox onChange={onChange} label={i18n.get('storeDetails')} name={'storeDetails'} />
```

Comment:
Shouldn't the previous `value` property be passed as: `checked: !!value`?
Doesn't it serve as a way to say whether the checkbox comes "pre-checked"?